package.json

```diff
@@ -20,11 +20,13 @@
     "liftoff": "3.1.0",
     "lodash": "^4.17.11",
     "mkdirp": "^0.5.1",
+    "oracledb": "^3.1.2",
```

Comment:
should it be prodDependency?

Reply:
nope, we don't have any of the drivers as knex deps to be able to install only one that is used in the project.

Reply:
should be moved then

Reply:
oracledb is still in prod dependencies :D